src/Row.js

```diff
@@ -46,7 +46,7 @@ var Row = React.createClass({
   render(): ?ReactElement {
     var className = joinClasses(
       'react-grid-Row',
-      "react-grid-Row--${this.props.idx % 2 === 0 ? 'even' : 'odd'}"
+      'react-grid-Row--' + this.props.idx % 2 === 0 ? 'even' : 'odd'
```

Comment:
A more simple solution would be to replace the double quotes `"` with backticks `` ` `` to make use of the new ES6 string interpolation